coralproject/talk
# CORL-1738: Posting Comment and Replying appear out of order

## What does this PR do?

- Fixes an issue were posting or replying would create a comment out of chronological order
- Refactor and smaller improvements

## How do I test this PR?
- Scroll until you're not tailing
- Now in another window post new comments
- Back to the original window, without scrolling down, post comment
- Scroll down and see your comment appear in correct order
- repeat for conversation view
